_posts/en/newsletters/2024-06-14-newsletter.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 'Bitcoin Optech Newsletter #307'
+permalink: /en/newsletters/2024/06/14/
+name: 2024-06-14-newsletter
+slug: 2024-06-14-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Draft BIP for quantum-safe address format:** developer Hunter Beast
+  [posted][beast post] to both Delving Bitcoin and the mailing list a
+  ["rough draft" BIP][quantum draft] for assigning version 3 segwit
+  addresses to a [quantum resistant][topic quantum resistant] signature
+  algorithm.  The draft BIP describes the problem and links to several
+  potential algorithms along with their expected onchain size.  The
+  choice of algorithms and the specific details of implementation are
+  left for future discussion, as are additional BIPs necessary to full
+  realize the vision of adding full quantum resistance to Bitcoin.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+[Don't wipe indexes again when continuing a prior reindex][review club
+30132] is a PR by [TheCharlatan][gh thecharlatan] that can improve
+startup time when a user restarts their node before an ongoing reindex
+has completed.
+
+Bitcoin Core implements five indexes. The UTXO set and the block index
+are required, whereas the transaction index, compact block filter index
+and coinstats index are optional. When running with `-reindex`, all
+indexes are wiped and rebuilt. This process can take quite a while, and
+it is not guaranteed to finish before the node is stopped for any
+reason.
+
+Because the node needs to have an up to date UTXO set and block index,
+the reindexing status is persisted on disk. When a reindex is started, a
+flag is [set][reindex flag set], and it will only be unset when the
+reindex is finished. This way, when the node starts, it can detect that
+it should continue reindexing, even if the user didn’t provide the flag
+as a startup option.
+
+When restarting (without `-reindex`) after an unfinished reindex, the
+required indexes are preserved and will continue to be rebuilt further.
+Prior to [Bitcoin Core #30132][], the optional indexes would be wiped a
+second time. [Bitcoin Core #30132][] can make node startup more
+efficient by avoiding the wiping of the optional indexes when it is not
+necessary.
+
+{% include functions/details-list.md
+  q0="What is the behaviour change introduced by this PR?"
```

Comment:
```suggestion
  q0="What is the behavior change introduced by this PR?"
```
(and elsewhere) ... I'm not sure if we want the British spelling, but we use the American spelling most often so far:
```
$ git grep -i behaviour | wc -l
29
$ git grep -i behavior | wc -l
286
$
```
Or maybe we don't have to be consistent!